src/tetris/brick/brick.ts

```diff
@@ -19,6 +19,26 @@ export default class Brick {
 	public get blocks(): Block[] {
 		return this._blocks;
 	}
+
+	public moveLeft(): void {
+		// TODO: Implement
+	}
+
+	public moveRight(): void {
+		// TODO: Implement
+	}
+
+	public drop(): void {
+		// TODO: Implement
+	}
+
+	public rotate(): void{
```

Comment:
In which direction should the brick be rotated? Clockwise vs Counter-Clockwise

Reply:
From my point of view common tetris game only allow one direction rotation (clockwise).